react-todo/src/components/utils/dbPost.ts

```diff
@@ -0,0 +1,18 @@
+import { ITodoItem } from "../../Interfaces/Interfaces";
+
+export const dbPost = async (item: ITodoItem) => {
```

Comment:
it's better to create a new service/ folder and extract all the server methods there. Also, don't forget about the typescript and specify the return types of the every function